parse.py

```diff
@@ -48,33 +52,51 @@ def ocr_space_url(url, overlay=False, api_key= ocr_key, language='eng', isTable=
     :param isTable: parses output as a table
     """
 
-    payload = {'url': url,
-               'isOverlayRequired': overlay,
-               'apikey': api_key,
-               'language': language,
-               'isTable': isTable,
-               }
-    r = requests.post('https://api.ocr.space/parse/image',
-                      data=payload,
-                      )
-    #return r.content.decode()
+    payload = {
+        "url": url,
+        "isOverlayRequired": overlay,
+        "apikey": api_key,
+        "language": language,
+        "isTable": isTable,
+    }
+    r = requests.post("https://api.ocr.space/parse/image", data=payload,)
+    # return r.content.decode()
+
+    # Change this return in source code to return JSON object, not string
     return r.json()
 
-data = ocr_space_url('https://ocr.space/Content/Images/receipt-ocr-original.jpg')
+#API response and obtaing "LineText"
+def parse():
+    words = []
+    data = ocr_space_url("https://ocr.space/Content/Images/receipt-ocr-original.jpg")
+    for i in data.get("ParsedResults")[0]["TextOverlay"]["Lines"]:
+        words.append(i.get("LineText"))
 
-#print(data.get('ParsedResults')[0]['TextOverlay']['Lines'][22].get('LineText'))
-words = []
+#Query the excel sheet and obtain the target food groups. Note not all used
+def FoodDatabase():
+    df = pd.read_excel("food.xlsx", usecols="B:C")
+    dairy = df.query("FdGrp_Cd == 100")["Long_Desc"].tolist()
```

Comment:
It would be more elegant here to use slicing syntax: https://stackoverflow.com/a/12098586

For example,

```
df[df['FdGrp_Cd'].isin(['100'])]['Long_Desc'].tolist()
```

This allows `df['FdGrp_Cd']` to be pulled out into a separate variable:

```
food_group_code = df['FdGrp_Cd']
df[food_group_code.isin(['100'])]['LongDesc'].tolist()
```

Also I think you can just do

```
df[food_group_code.isin(['100']), 'LongDesc'].tolist()
```

Reply:
I really thought the last line was the right way but it didn't work. I think it's because we are filtering it first, then calling `Long_Desc` instead of just calling the respective headers. 

After numerous times, i just went with `df[food_group_code.isin(['100'])]['LongDesc'].tolist()`